qutebrowser/config/configdata.py

```diff
@@ -1638,8 +1638,8 @@ def data(readonly=False):
         ('move-to-end-of-line', ['$']),
         ('move-to-start-of-document', ['gg']),
         ('move-to-end-of-document', ['G']),
-        ('yank-selected -p', ['Y']),
-        ('yank-selected', ['y'] + RETURN_KEYS),
+        ('yank selection -p', ['Y']),
```

Comment:
Since we'll doing some migrations here anyways, probably makes sense to fix that up at the same time.